x-pack/test_serverless/functional/test_suites/common/platform_security/navigation/management_nav_cards.ts

```diff
@@ -63,7 +63,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
     describe('as viewer', function () {
       before(async () => {
-        await pageObjects.svlCommonPage.loginWithRole('viewer');
+        await pageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Same as in the [avatar menu test](https://github.com/elastic/kibana/pull/188128/files?file-filters%5B%5D=.ts&owned-by%5B%5D=jeramysoucy&show-viewed-files=true#r1681885420) - should log in as viewer, not editor/developer

Reply:
https://github.com/elastic/kibana/pull/188128/commits/badde66df75dea281bfa8a182ee1158b664eb894